ElectronNET.API/App.cs

```diff
@@ -1244,7 +1396,14 @@ internal void PreventQuit()
         /// <param name="fn">The handler</param>
         public void Once(string eventName, Action<object> fn) => Events.Instance.Once(ModuleName, eventName, fn);
 
-        private readonly JsonSerializer _jsonSerializer = new JsonSerializer()
+
+        /// <summary>
+        /// If you're using a splashscreen in the electron.manifest.json, the window will ony be fully destroyed once you call this method once.
+        /// You should only do this after creating another window, to avoid a bug where the Electron renderer process frezees till any window interaction.
+        /// </summary>
+        public void DestroySplashScreen() =>  BridgeConnector.Emit("splashscreen-destroy");
```

Comment:
This should definitely be called out in any documentation as a new unexpected requirement or something.